Cython/Compiler/Symtab.py

```diff
@@ -2509,6 +2525,9 @@ def declare_inherited_cpp_attributes(self, base_class):
                     base_entry.type, None, 'extern')
                 entry.is_variable = 1
                 entry.is_inherited = 1
+                entry.is_cfunction = base_entry.is_cfunction
+                if entry.is_cfunction:
+                    entry.func_cname = base_entry.func_cname
```

Comment:
I suggest making the whole thing depend on `if base_entry.is_cfunction`.

Reply:
Done, thanks for the suggestion.